### DISCUSSION TITLE
How to sell rare and exotic sats?

### DISCUSSION THREAD

**Discussion by earthflower** [ordinals/ord#1605] [Q&A]
I need a listing mechanism to sell rare and exotic sats

**Comment by casey** [comment]
Currently all trades are P2P and fully trusted. We have ideas to improve this planned, but for now, trade at your own risk!

**Comment by Blockchainwolf** [comment]
Currently, you can sell them on twitter with the hashtag #Ordinals. Few buyers i talked with is hunting such sellers through twitter and discord groups. 

however i will suggest you to hold your exotic till the team comeup with marketplace

**Comment by reinamora137** [comment]
ordsea.io and ordswap.io are a couple upcoming options for listing inscriptions. No affiliation. 

ordsea you maintain the inscriptions in your wallet and it is OTC - you just verify ownership there and list.

ordswap you send them your inscription to your ordswap.io wallet. Kind of like a CEX.
